src/organisms/cards/PlaybackCard/index.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Partial, separatePartials, partialRenderer } from 'helpers/Partial';
+import ImageAside from 'molecules/asides/ImageAside';
+import Layout from 'atoms/Layout';
+import Col from 'atoms/Layout/Col';
+import LinkWrapper from 'atoms/LinkWrapper';
+
+import PlaybackCardPartials from './PlaybackCardPartials';
+import styles from './playback_card.module.scss';
+
+function PlaybackCard(props) {
+  const {
+    children,
+    header,
+    onClick
+  } = props;
+
+  const parts = separatePartials(children);
+  const renderPartial = partialRenderer(PlaybackCardPartials, styles.playbackCard);
```

Comment:
i'm unclear why you are passing `styles.playbackCard` as a second arg? it seems `partialRenderer` only takes one